### PR TITLE
Evildoer98/dailylearning

### PR DESCRIPTION
**Pull request by Evildoer98**
# time: 22/3/7

**What kind of change does this PR introduce?** (check at least one)

- [x] notes: Notes
- [x] codes: Codes
- [ ] leetcode: LeetCode
- [ ] demo: Demo
- [ ] feature: Feature

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing codes:

**Describe the change introduced by this PR:**

**Content:**
- [x] reactive

**Related links**
- [x] Document from notion: https://v3.cn.vuejs.org/api/basic-reactivity.html#reactive

**Other information:**

### CODE REVIEW

File: daily-js/闭包/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  return function(b) {
+    return a+b
+  }
+}
+
+var add5 = makeAdder(5)
+var add20 = makeAdder(20)
+
+add5(6) // 11
+add20(7) // 27
```
Comment (LaicZhang) [inline daily-js/闭包/index.js:11]: 说好的多写点闭包应用场景喃，真不写嗷，可恶

File: daily/day_02-事件循环/宏任务微任务 2.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## 宏任务和微任务
+
+- 微任务包括：MutationObserver、Promise.then()或catch()、Promise为基础开发的其它技术，比如fetch API、V8的垃圾回收过程、Node独有的process.nextTick。
+
+- 宏任务包括：script 、setTimeout、setInterval 、setImmediate 、I/O 、UI rendering。
```
Comment (LaicZhang) [inline daily/day_02-事件循环/宏任务微任务 2.md]: 好像还有一个`requestAnimationFrame `

File: daily/day_07——v-if和v-for,不能同时使用?/day_07 2.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ * @Author: Evildoer98
+ * @Date: 2021-10-18 21:13:59
+ * @LastEditors: Evildoer98
+ * @LastEditTime: 2021-10-18 21:15:02
+-->
+
+vue 中，v-if 和 v-for 不能同时使用？
+
+原因：
+    1. v-for 优先级高
```
Comment (LaicZhang) [inline daily/day_07——v-if和v-for,不能同时使用?/day_07 2.md]: vue3中v-if优先级高

File: daily-js/闭包/index.js (hunk 1 above)
Comment (Evildoer98) [reply daily-js/闭包/index.js:11]: 不急嘛，后面再写，现在没啥时间

File: daily/day_07——v-if和v-for,不能同时使用?/day_07 2.md (hunk 3 above)
Comment (Evildoer98) [reply daily/day_07——v-if和v-for,不能同时使用?/day_07 2.md]: 在 vue2 中，v-for 的优先级高
在 vue3 中，v-if 的优先级高

文档：https://v3.cn.vuejs.org/guide/migration/v-if-v-for.html#frontmatter-title

File: daily/day_02-事件循环/宏任务微任务 2.md (hunk 2 above)
Comment (Evildoer98) [reply daily/day_02-事件循环/宏任务微任务 2.md]: 对

**Review by LaicZhang — APPROVED** [review]

**Pull request merged**